### ISSUE TITLE
versionCode not increased?

### ISSUE THREAD

**Issue by IzzySoft** [mkckr0/audio-share#2]
Looks like you forget to increase `versionCode` with each new release:

    package: name='io.github.mkckr0.audio_share_app' versionCode='1' versionName='0.0.4'

`versionCode` is what Android uses internally to tell versions apart – so same `versionCode` means same version to it and the new APK won't be considered an update. Could you please take care to [have it increased](https://github.com/mkckr0/audio-share/blob/main/audio-share-app/app/build.gradle#L15) whenever you prepare a release? Thanks in advance!

**Comment by IzzySoft** [comment]
Oof. And now comes the next release, still the same `versionCode` – and signed by another signing key? I'm quite confused…

**Comment by mkckr0** [comment]
Yes, the `versionCode` should be increased. I'll increase it in the future release. For signing key, I move the release procedure to Github Action, which uses a different debug signing key. I'll delete this release and then test it.

**Comment by IzzySoft** [comment]
Thanks @mkckr0 – but should releases be signed by a debug key? And doesn't this further mean the key must be available at Github?

**Comment by mkckr0** [comment]
I have finished the Github Action config, and released a new version by Github Action.
https://github.com/mkckr0/audio-share/releases/tag/v0.0.5
I save release.jks to this repo's secret variable, and will use it for future releases.

**Comment by IzzySoft** [comment]
Hi @mkckr0 – and congrats to your listing at F-Droid.org! As your app is available there now for 2 weeks, this would be the time I'd remove it from my repo. But I see your Readme still carries the badge linking here. Does that mean you want your app to stay in my repo, too? As you've established reproducible builds with F-Droid, and the APK is decently sized, that would be possible (though I try to keep the overlap minimal as my repo still runs on "limited" personal space).

**Comment by mkckr0** [comment]
Ok. I'll remove the badge. Thanks for your guidance.

**Comment by IzzySoft** [comment]
Thanks! Should you one day need the app back in my repo for some reason, just let me know. Metadata have been archived, so restore is easy.
